package.json

```diff
@@ -35,8 +35,8 @@
     "js-lint": "eslint js/ && eslint --config js/tests/.eslintrc.json --env node build/",
     "js-lint-docs": "eslint --config js/tests/.eslintrc.json assets/js/ docs/ sw.js",
     "js-compile": "npm-run-all --parallel js-compile-*",
-    "js-compile-standalone": "cross-env ROLLUP=true rollup --environment BUNDLE:false --config build/rollup.config.js --sourcemap",
-    "js-compile-bundle": "cross-env ROLLUP=true rollup --environment BUNDLE:true --config build/rollup.config.js --sourcemap",
+    "js-compile-standalone": "cross-env rollup --environment BUNDLE:false --config build/rollup.config.js --sourcemap",
```

Comment:
Does this mean we can get rid of `cross-env` in these 2 lines?

Reply:
Yep thanks to the removal of external-helpers

Reply:
Can you remove it where it's no longer needed?

Reply:
Nope we still use it one line under for `js-compile-plugins` instead of if we find a way to avoid the use of `cross-env`

Reply:
Yes, but it seems it's redundant in these 2 lines here (not the plugins script

Reply:
Yep I'll remove it